src/injector.js

```diff
@@ -1,8 +1,10 @@
 import { Bottle } from 'bottlejs';
 
+import { DEFAULT_SCOPE } from './constants';
 
-export function getInjector(scope = null) {
-    const key = scope || 'nodule';
+
+export function getInjector(scope = DEFAULT_SCOPE) {
+    const key = scope || DEFAULT_SCOPE;
```

Comment:
There's still an edge case here where explicitly passing `null` will use the default scope. I'm ok with this.